tools/buildutils/build_packages.sh

```diff
@@ -25,7 +25,11 @@ function build_package() {
 REPO_DIR="$(realpath "$(dirname "$0")/../..")"
 INSTALL_BAZEL="$(dirname $0)/installbazel.sh"
 
-sudo "${INSTALL_BAZEL}"
+if command -v bazel &> /dev/null; then
```

Comment:
let's rather use the OR (||) having a single self explanatory line: `command -v bazel &> /dev/null || sudo "${INSTALL_BAZEL}"`

Reply:
sure.

Reply:
I have do rebase and push again. Please help to check if meets the submission requirement.